server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const app = express()
+
+
+//routes
+
+app.get('/', (req, res) => {
+    res.sendStatus("Hello NODE API")
+})
+
+
+app.listen(5100, ()=> {
```

Comment:
Consider using environment variables for the port number instead of hardcoding it. This will make your application more flexible and environment agnostic. [important]